doc/geany.txt

```diff
@@ -364,8 +364,8 @@ General
 -------
 
 
-Startup
-^^^^^^^
+Re-opening files
+^^^^^^^^^^^^^^^^
```

Comment:
This section also documents opening recently used files, so I renamed the title.